### PR TITLE
Create javax-inject_ubuntu_20.04.sh

### PR DESCRIPTION
**Pull request by gururajrkatti**
build script for javax-inject

### CODE REVIEW

File: j/javax-inject/javax-inject_ubuntu_20.04.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#!/bin/bash
+
+sudo apt-get update
+sudo apt-get install openjdk-8-jdk wget git -y
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-ppc64el/
+
+wget http://www-eu.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz
+tar xzvf apache-maven-3.6.3-bin.tar.gz
+export PATH=$PATH:`pwd`/apache-maven-3.6.3/bin
+git clone https://github.com/javax-inject/javax-inject
```
Comment (seth-priya) [inline j/javax-inject/javax-inject_ubuntu_20.04.sh:26]: I believe the reason we are going with top-of-tree/master is that this project does not have a current release or tag that we can use, is that so?

Comment (gururajrkatti) [reply j/javax-inject/javax-inject_ubuntu_20.04.sh:26]: Yes @seth-priya it doesn't have any release tags

**Comment by seth-priya** [comment]
@gururajrkatti also, we need the Apache 2.0 license file to go along with each script

**Comment by gururajrkatti** [comment]
@seth-priya I have added the license file. Pl. check

**Review by seth-priya — APPROVED** [review]

**Pull request merged**